api/CanvasRenderingContext2D.json

```diff
@@ -2125,15 +2125,21 @@
                 "prefix": "webkit"
               }
             ],
-            "edge": {
-              "version_added": "15"
-            },
+            "edge": [
```

Comment:
Can you update the title description to cover Edge?

Reply:
Didn't mean to include Edge changes here, oops!